runtime/lua/vscode-neovim/internal.lua

```diff
@@ -268,4 +268,15 @@ function M.start_visual(buf, anchor, active)
   api.nvim_feedkeys((v == "V" or v == "\x16") and "gvv" or "gv", "n", false)
 end
 
+---Translate from a Windows path to a WSL path
+---@param path string
+---@return string?
+function M.wslpath(path)
```

Comment:
Why do we need this, can't we use the ts wsl function?

Reply:
Executing wsl.exe creates a new connection to the instance each time, which is unnecessary. Since nvim is already running in the instance, it is more efficient and reasonable to use it as a bridge to execute Linux commands.